src/custom/constants/chainInfo/chainInfoMod.ts

```diff
@@ -90,17 +90,17 @@ export const CHAIN_INFO: ChainInfoMap = {
     label: 'Kovan',
     logoUrl: ethereumLogoUrl,
     nativeCurrency: { name: 'Kovan Ether', symbol: 'kovETH', decimals: 18 },
-  },
+  }, */
   [SupportedChainId.GOERLI]: {
     networkType: NetworkType.L1,
-    docs: 'https://docs.uniswap.org/',
+    docs: 'https://docs.cow.fi/',
     explorer: 'https://goerli.etherscan.io/',
-    infoLink: 'https://info.uniswap.org/#/',
+    infoLink: 'https://cow.fi/',
     label: 'Görli',
-    logoUrl: ethereumLogoUrl,
+    logoUrl: RinkebyLogo, // TODO: should we change this
```

Comment:
Ofcourse we should :) 

If there's no official icon, we could do as in rinkebym but use the blue color instead of yellow 

we use Metamasks colors

<img width="459" alt="image" src="https://user-images.githubusercontent.com/2352112/170548597-c2bc83f1-a975-4dfa-8c9a-b553d2950a77.png">

Reply:
Added logo